preset/variables.scss

```diff
@@ -144,6 +144,9 @@ $dialog-border-radius: 8px;
 // v-input
 $input-label-letter-spacing: normal;
 
+// textarea
+$textarea-line-height: 1.5rem;
```

Comment:
I wonder if this should be `1.5` (e.g. 150% of the text area's font size) rather than `1.5rem` which is effectively 150% of the _root element_ font size?

I'm assuming Andrew would expect the line height to scale on the off chance we have other text areas with different font sizes later.

Reply:
Hmm I do not know. In the Vuetify docs it has `$textarea-line-height | 1.75rem !default;`.

Shall I just make it `1.5`?

Hmm, this is what is in the jira
`Response input - the line height should be 1.5x the font size. ie. 24px line height on 16px font size. At the moment it is 1.75rem.`
So yes think you are right, he does want it 1.5 the font size.

Cool, will make that change.